app/assets/javascripts/va_facilities/facilities_utilities.es6

```diff
@@ -0,0 +1,139 @@
+const visnSelect = "#facility_visn_select";
```

Comment:
⚠️ moved the above file `app/assets/javascripts/facilities_utilities.es6` to a folder to match the structure of other es6 files